readme.md

```diff
@@ -266,6 +266,10 @@ setTimeout(() => controller.abort(), 5000);
 })();
 ```
 
+### Method name normalization
+
+Ky normalizes the common used method names (`GET`, `POST`, `PATCH`, etc.) in order to avoid server errors due to case sensitivity.
```

Comment:
I was thinking just documenting the `method` option here: https://github.com/sindresorhus/ky#options And mentioning the normalization we do.